flight/mcu/lib/teensyShared/GPSSensor.cpp

```diff
@@ -1,33 +1,71 @@
 #include "GPSSensor.h"
 
-#include <ChRt.h>
-#include <SparkFun_u-blox_GNSS_Arduino_Library.h>
+#include "dataLog.h"
+#include "pins.h"
 
-void GPSSensor::readReadings() { GNSS->getPVT(20); }
 
-float GPSSensor::getLatitude() {
-    latitude = GNSS->getLatitude();
+GPSSensor gps;
+
+void GPSSensor::init() {
+    SPI1.begin();  // TODO should this line be moved?
```

Comment:
Now that the init code isn't in the main setup function anymore I don't think stalling the program is an acceptable failure mode. Init should return a boolean indicating success or failure and main can decide what to do with it.